stdlib/REPL/src/REPL.jl

```diff
@@ -1479,11 +1479,17 @@ ends_with_semicolon(code::AbstractString) = ends_with_semicolon(String(code))
 ends_with_semicolon(code::Union{String,SubString{String}}) =
     contains(_rm_strings_and_comments(code), r";\s*$")
 
-function banner(io::IO = stdout; short = false)
-    if Base.GIT_VERSION_INFO.tagged_commit
-        commit_string = Base.TAGGED_RELEASE_BANNER
+"""
+    banner(io::IO = stdout, maxsize:Int = 10)
+
+Print the "Julia" informative banner to `io`, using a size variant no larger
+than `maxsize`.
```

Comment:
What is size measured in? Is it rows/why custom rows and not custom columns?

Reply:
So, I'll need to elaborate obviously but it's an ordinal index of banner sizes, with `1` as the smallest and going up from there.